### PR TITLE
Submissions for pattern catalog

### PR DESCRIPTION
**Pull request by b0id**
**Description**

This PR fixes #

Submission protocol for catalog.

**Notes for Reviewers**

Some local rendering issue with the video even for code that I didn't add or modify, must be reviewed and revised.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by b0id** [comment]
Web view of video renders fine.  Still need to update screenshot for video preview, may revise video.

**Comment by b0id** [comment]
I have a couple of things I would like to tweak, but overall I think this is sufficient, pending review.

File: _includes/partials/callouts-platform-choice.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="callout2">
+    <a class="card2" href="https://meshery.io/#getting-started" target="_self">
+      <h3>Meshery MeshMap</h3>
```
Comment (Chadha93) [inline _includes/partials/callouts-platform-choice.html:3]:
```suggestion
      <h3>MeshMap</h3>
```

File: _includes/partials/callouts-platform-choice.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+      <p class="small">Use Meshery MeshMap from a platform of your choice. <br><br> You will be redirected, to the getting started section. Pick your favorite platform to deploy Meshery.</p>
```
Comment (Chadha93) [inline _includes/partials/callouts-platform-choice.html:4]:
```suggestion
      <p class="small">Use MeshMap from a platform of your choice. <br><br> You will be redirected, to the getting started section. Pick your favorite platform to deploy Meshery.</p>
```

**Review by Chadha93 — APPROVED** [review]
COOL, LGTM

**Review by Chadha93 — APPROVED** [review]

**Pull request merged**